content/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Overview"
+date: 2021-11-15T15:06:19-05:00
+draft: false
+---
+
+# What is crane?
+
+Crane is a tool that helps application owners migrate workloads between 4.x clusters. It is a tool that is divided in four parts.
+
+- [Crane-cli](https://github.com/konveyor/crane): This is the cli(front end) of the process which allows user perform every action required.
```

Comment:
So the cli's name is just "crane". Maybe we should just use "crane" in the link name instead of "Crane-cli" -- the description makes it clear that it's a cli.